clients/cli/src/main/java/org/apache/gravitino/cli/GravitinoCommandLine.java

```diff
@@ -175,6 +176,10 @@ private void handleTableCommand() {
     if (CommandActions.DETAILS.equals(command)) {
       String table = name.getTableName();
       new TableDetails(url, ignore, metalake, catalog, schema, table).handle();
+
+      if (line.hasOption(GravitinoOptions.INDEX)) {
+        new ListIndexes(url, ignore, metalake, catalog, schema, table).handle();
+      }
```

Comment:
This should be an if-else, only details or indexes should be displayed, not both.